test/jdk/jdk/internal/platform/docker/MetricsMemoryTester.java

```diff
@@ -73,11 +73,11 @@ private static void testMemoryFailCount() {
             long count = Metrics.systemMetrics().getMemoryFailCount();
 
             // Allocate 512M of data
```

Comment:
Suggestion: Amend the comment to `Allocate 512M of data in 1M chunks per iteration`